src/extensibility/ExtensionManagerDialog.js

```diff
@@ -420,8 +424,11 @@ define(function (require, exports, module) {
                 });
             });
             
-            // Open dialog to Installed tab if extension updates are available
-            if ($("#toolbar-extension-manager").hasClass('updatesAvailable')) {
+            var $activeTab = $dlg.find(".nav-tabs li.active a");
+            if ($activeTab.length) { // If there's already a tab selected, show it
+                $activeTab.tab("show");
```

Comment:
Rather than making changes to the bootstrap-tab widget, you could put this line of code above the show statement:
```
$activeTab.parent().removeClass("active");
```

I think this is cleaner because this way you don't have to modify bootstrap which could be hard to maintain and could have side effects for other parts of the app.